# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/ipc/DocAccessibleParent.h
# Commit: 16b49e8c315b
# Full Hash: 16b49e8c315bb512c0bc406670d3b4be653185b4
# Author: James Teh <jteh@mozilla.com>
# Date: 2023-11-30 04:12:04
# Regressor Bug: 1862802
# File Overlap Count: 1
# Description:
#   Bug 1862802 part 5: Remove virtual cursor change event. r=eeejay
#   
#   Depends on D192645
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D192646
# ==============================================================================

diff -r 5a6669e44c53 -r 16b49e8c315b accessible/ipc/DocAccessibleParent.h
--- a/accessible/ipc/DocAccessibleParent.h	Wed Nov 29 03:53:20 2023 +0000
+++ b/accessible/ipc/DocAccessibleParent.h	Wed Nov 29 03:53:20 2023 +0000
@@ -125,11 +125,6 @@
       const uint64_t& aID, const uint64_t& aWidgetID,
       const uint32_t& aType) override;
 
-  virtual mozilla::ipc::IPCResult RecvVirtualCursorChangeEvent(
-      const uint64_t& aID, const uint64_t& aOldPositionID,
-      const uint64_t& aNewPositionID, const int16_t& aReason,
-      const bool& aFromUser) override;
-
   virtual mozilla::ipc::IPCResult RecvScrollingEvent(
       const uint64_t& aID, const uint64_t& aType, const uint32_t& aScrollX,
       const uint32_t& aScrollY, const uint32_t& aMaxScrollX,